src/sentry/incidents/action_handlers.py

```diff
@@ -179,3 +127,63 @@ def format_duration(minutes):
 
     seconds = minutes / 60
     return "{} second{}".format(seconds, pluralize(seconds))
+
+
+INCIDENT_STATUS_KEY = {
+    IncidentStatus.OPEN: "open",
+    IncidentStatus.CLOSED: "resolved",
+    IncidentStatus.CRITICAL: "critical",
+    IncidentStatus.WARNING: "warning",
+}
+
+
+def generate_incident_trigger_email_context(project, incident, alert_rule_trigger, status):
```

Comment:
Why'd we move this out to a separate function?

Reply:
Needed it for the debug, but apparently I just didn't look hard enough